rust/server/src/main.rs

```diff
@@ -23,11 +23,32 @@ use api::kv_store::KvStore;
 use impls::postgres_store::PostgresBackendImpl;
 use std::sync::Arc;
 
+pub(crate) mod util;
 pub(crate) mod vss_service;
 
 fn main() {
-	// Define the address to bind the server to
-	let addr = SocketAddr::from(([127, 0, 0, 1], 3000));
+	let args: Vec<String> = std::env::args().collect();
```

Comment:
nit: Instead of collecting we can simply use the `Iterator` interface implemented by `Args`.